patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -207,6 +221,33 @@
 +    protected ThreadLocal<EntityPlayer> harvesters = new ThreadLocal();
 +    private ThreadLocal<IBlockState> silk_check_state = new ThreadLocal();
 +    protected static java.util.Random RANDOM = new java.util.Random(); // Useful for random things without a seed.
++
++    /**
++     * Called for ref-counting purposes by ExtendedBlockStorage in order to
```

Comment:
This is not the main purpose of `getTickRandomly`, it's a secondary purpose.
Please update the javadoc with a better description.